custom/icds_reports/reports/cas_reach_data.py

```diff
@@ -101,7 +110,7 @@ def get_data_for_daily_usage(date, filters):
                     'help_text': awcs_launched_help_text(),
                     'color': None,
                     'percent': None,
-                    'value': get_value(daily_yesterday if current_month_selected else awc_this_month_data, 'awcs'),
+                    'value': get_value(awc_this_month_data if pre_release_features else daily_yesterday, 'awcs'),
```

Comment:
it should be more like
`daily_yesterday if pre_release_features and current_month_selected else awc_this_month_data`
@dhiraj-beehyv